best-practices/README.md

```diff
@@ -96,6 +96,7 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryBot factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Use `app/models` for model objects and `app/services` for command objects.
```

Comment:
"model" means different things to different people. Depending on who you ask, a model might be only objects that inherit from Active Record or it might be any domain object.

Reply:
I'll change this to say "domain object" instead of "model object".